packages/simple-hub/.env.development

```diff
@@ -1,5 +1,5 @@
-FIREBASE_PROJECT='rock-paper-scissors-dev-4ff8d'
```

Comment:
Where is this env var intended to be used?

Reply:
Maybe on staging?